src/features/store.ts

```diff
@@ -114,6 +114,8 @@ export const useStore = create<AppState>()(
           txs[i].status = s;
           txs[i].msgId ||= options?.msgId;
           txs[i].originTxHash ||= options?.originTxHash;
+          txs[i].orderId ||= options?.orderId;
+          txs[i].remoteTxHash ||= options?.remoteTxHash;
```

Comment:
two extra attributes added to the tx information. This will help to track the intent completion and to host the transfer information on the destination chain.

<img width="477" alt="image" src="https://github.com/user-attachments/assets/59983768-2257-4031-a966-eda731e68600" />